### PR TITLE
Add more documentation on how to replace g_signal_connect_object

### PR DESCRIPTION
**Pull request by pbor**

### CODE REVIEW

File: glib/src/object.rs [hunk 1]
```diff
@@ -1834,6 +1834,7 @@ pub trait ObjectExt: ObjectType {
 
     // rustdoc-stripper-ignore-next
     /// Similar to [`Self::connect_closure`] but fails instead of panicking.
+    #[doc(alias = "g_signal_connect_object")]
```
Comment (bilelmoussaoui) [inline glib/src/object.rs:1837]: should probably be added on connect_closure as well? :)

File: glib/src/object.rs [hunk 2]
```diff
@@ -1858,7 +1865,6 @@ pub trait ObjectExt: ObjectType {
     ) -> SignalHandlerId;
 
     /// Similar to [`Self::connect_closure_id`] but fails instead of panicking.
-    #[doc(alias = "g_signal_connect_closure")]
```
Comment (sdroege) [inline glib/src/object.rs:1861]: Why? I would add both aliases here too

Comment (pbor) [reply glib/src/object.rs:1861]: I can add them back, but imho it was weird to have them on the `try_` variant

Comment (sdroege) [reply glib/src/object.rs:1861]: Why? It does exactly the same, just returns a result instead of panicking

Comment (pbor) [reply glib/src/object.rs:1861]: well, then we should add them to all the other `try` variants as well, this was the one-off... I can do that

Comment (sdroege) [reply glib/src/object.rs:1861]: Oh I see. Then let's not do that for now and add them all at once later if needed. Thanks for noticing that :)

**Review by sdroege — APPROVED** [review]

**Pull request merged**